packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/add/add.routing.js

```diff
@@ -36,6 +36,23 @@ export default /* @ngInject */ ($stateProvider) => {
           );
         }),
       goBack: /* @ngInject */ (goToStorageContainers) => goToStorageContainers,
+      encryptionAlgorithms: /* @ngInject */ ($http, encryptionAvailable) =>
+        encryptionAvailable
+          ? $http
+              .get('/cloud.json')
+              .then(
+                ({ data: { models } }) =>
+                  models['cloud.storage.EncryptionAlgorithmEnum']?.enum || [],
+              )
+          : [],
```

Comment:
IMO, this should not fail silently
By returning an empty array, the only thing the user will be displayed is just an introduction text with nothing to chose and no message to warn him that something went wrong

Reply:
fixed.

I added fallback values to ensure the availability of the service